build.sbt

```diff
@@ -1,133 +1,44 @@
-import ReleaseTransformations._
 import microsites.ExtraMdFileConfig
 import microsites.ConfigYml
 import sbtcrossproject.{ CrossProject, CrossType }
-import scala.xml.{ Elem, Node => XmlNode, NodeSeq => XmlNodeSeq }
-import scala.xml.transform.{ RewriteRule, RuleTransformer }
 
-def Scala3 = "3.1.0"
+ThisBuild / tlBaseVersion := "0.14"
+ThisBuild / tlCiReleaseTags := false
```

Comment:
This disables CI release, but say the word (and configure the [secrets](https://typelevel.org/sbt-typelevel/secrets.html)) and I'll switch it on :)